views/FanView/FanView.tsx

```diff
@@ -10,12 +10,21 @@ import {
 } from '../../.storybook/stories';
 import { selectDeviceWithId } from '../../redux/devicesSlice';
 import { useAppSelector } from '../../redux/hooks';
-import { Fan, RootStackParamList } from '../../types';
+import { Fan, FanRangeValues, RootStackParamList } from '../../types';
 import { sendAPIRequest } from '../../utils';
 import { styles } from './FanView.styles';
 
 type FanViewProps = NativeStackScreenProps<RootStackParamList, 'Fan'>;
 
+const fanRangeValues: FanRangeValues = {
```

Comment:
Good idea to move it to const :D however, the problem with a const object is that in the JS const isn't a constant, it only means that the variable cannot be re-assigned. It means that you cannot change the whole object, but you can change the value of a particular properties
![image](https://github.com/inzynierka-home-iot/frontend-rncli/assets/48959539/ee83647d-1f60-4230-be1a-c6de2982fe9a)
I would consider using a enum or moving it to a separate const (remember to use the correct letter case - for enum PascalCase, for const CONST_CASE)

Reply:
I know about that. Changes applied.